cmd/internal-ip/main.go

```diff
@@ -1,63 +1,87 @@
 package main
 
 import (
+	"errors"
```

Comment:
This package won't be needed anymore in case you agree to apply my suggestions.

Reply:
Sure